docs/03_server/04_event-handler/03_advanced.md

```diff
@@ -2,11 +2,12 @@
 title: 'Event Handler - advanced'
 sidebar_label: 'Advanced'
 id: advanced
-keywords: [server, event handler, advanced]
+keywords: [server, event handler, advanced, pending approvals]
```

Comment:
not sure how search works but maybe pending, approvals (so approvals on its own is found)

Reply:
not sure how it works either, @wjhendry do you know more details?